zanata-war/src/main/webapp/WEB-INF/layout/admin/contact_admin_modal.xhtml

```diff
@@ -32,10 +34,26 @@
       return getContactAdminForm().find('#contact-admin-send-button');
     }
 
+    // Returns a span.
+    function getContactAdminMessageErrorField(){
+      return getContactAdminForm().find('#contactAdminForm\\:messageField\\:input\\:contact-admin-error-message\\:contactAdminForm\\:messageField\\:input\\:contact-admin-message');
```

Comment:
This seems like a very long Id. Is there a better way to find it?

You could use Richfaces' `rich:componentId('contact-admin-message')` which would return the whole client side id for the jsf component. After that it could be passed to the `getContactAdminForm().find(...)` method.